.goreleaser.yml

```diff
@@ -26,6 +26,7 @@ archives:
       linux: linux
       windows: windows
       freebsd: freebsd-experimental
+      openbsd: openbsd-experimental
```

Comment:
you also want this in `goos` section above. test with `make goreleaser`

Reply:
`make goreleaser` fails due to `goreleaser` not having an OpenBSD release package :)